lib/event_tracer/dynamo_db_logger.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'time'
+
+module EventTracer
+  class DynamoDBLogger
+    def initialize(buffer = nil)
+      @buffer = buffer
+    end
+
+    EventTracer::LOG_TYPES.each do |log_type|
+      define_method log_type do |**args|
+        save_message log_type, **args
+      end
+    end
+
+    private
+
+      attr_reader :buffer
+
+      def save_message(log_type, action:, message:, **args)
+        payload = prepare_payload(log_type, action: action, message: message, args: args)
+
+        if buffer
```

Comment:
Instead of having no buffer, could we have a buffer with size 0 by default? It would help shorten the code here

Reply:
Oh yea that's true. Will clean up the specs too then